packages/forms-web-app/src/dynamic-forms/dynamic-components/multi-field-input/question.js

```diff
@@ -39,7 +49,8 @@ class MultiFieldInputQuestion extends Question {
 		html,
 		label,
 		inputAttributes = {},
-		inputFields
+		inputFields,
+		formatType
```

Comment:
think references to formatType can go now? Otherwise looks good

Reply:
Yep, good spot!